misaligned.c

```diff
@@ -16,6 +16,7 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    assert(result == 0);

```

Comment:
Asserts are supposed to state the functionality of the code. This assert will fail even when the code is correct.